src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -767,23 +760,47 @@ private[snowflake] class SnowflakeSQLStatement(
       case (element, index) =>
         element match {
           case ele: StringVariable =>
-            statement.setString(index + 1, ele.variable)
+            if (ele.variable.isDefined)
+              statement.setString(index + 1, ele.variable.get)
+            else
+              statement.setNull(index + 1, java.sql.Types.VARCHAR)
           case ele: Identifier =>
-            statement.setString(index + 1, ele.variable)
+              statement.setString(index + 1, ele.variable.get)
```

Comment:
There is no "if (ele.variable.isDefined)" check for Identifier. Is it expected?

Reply:
Yes, it's expected. Identifier() does not take an option, because it's expected that it will never be None for Identifiers.